src/utils/ethereumUtils.ts

```diff
@@ -150,24 +168,24 @@ export const useEth = () => {
 };
 
 export const useEthUSDPrice = () => {
+  // @ts-expect-error ts-migrate(2339) FIXME: Property 'data' does not exist on type 'DefaultRoo... Remove this comment to see the full error message
   return useSelector(({ data: { ethUSDPrice } }) => ethUSDPrice);
 };
 
 export const useEthUSDMonthChart = () => {
+  // @ts-expect-error ts-migrate(2339) FIXME: Property 'charts' does not exist on type 'DefaultR... Remove this comment to see the full error message
   return useSelector(({ charts: { chartsEthUSDMonth } }) => chartsEthUSDMonth);
 };
 
-const getPriceOfNativeAssetForNetwork = network => {
-  return network === networkTypes.polygon
-    ? getMaticPriceUnit()
-    : getEthPriceUnit();
+const getPriceOfNativeAssetForNetwork = (network: Network) => {
+  return network === Network.polygon ? getMaticPriceUnit() : getEthPriceUnit();
 };
 
 const getEthPriceUnit = () => getAssetPrice();
 
 const getMaticPriceUnit = () => getAssetPrice(MATIC_MAINNET_ADDRESS);
 
-const getBalanceAmount = (selectedGasFee, selected) => {
+const getBalanceAmount = (selectedGasFee: any, selected: any) => {
```

Comment:
Any types here. I think `selectedGasFee` can be `LegacySelectedGasFee` and `selected` can be `ParsedAddressAsset`, or partials of those.